rogue-thi-app/next.config.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
             value: `default-src 'none';
               img-src 'self' data: https://tile.openstreetmap.org;
               font-src 'self';
-              connect-src 'self' wss://proxy.neuland.app ${DEEPL_ENDPOINT};
+              connect-src 'self' wss://proxy.neuland.app https://hiplan.thi.de ${DEEPL_ENDPOINT};
```

Comment:
can we use an environment variable here? Since we are planning to introduce a mock API server this hard-coding seems kind of wrong